sdk/servicebus/Microsoft.Azure.Management.ServiceBus/src/Generated/EventHubsOperationsExtensions.cs

```diff
@@ -1,99 +0,0 @@
-// <auto-generated>
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for
-// license information.
-//
-// Code generated by Microsoft (R) AutoRest Code Generator.
-// Changes may cause incorrect behavior and will be lost if the code is
-// regenerated.
-// </auto-generated>
-
-namespace Microsoft.Azure.Management.ServiceBus
-{
-    using Microsoft.Rest;
-    using Microsoft.Rest.Azure;
-    using Models;
-    using System.Threading;
-    using System.Threading.Tasks;
-
-    /// <summary>
-    /// Extension methods for EventHubsOperations.
-    /// </summary>
-    public static partial class EventHubsOperationsExtensions
-    {
-            /// <summary>
-            /// Gets all the Event Hubs in a service bus Namespace.
-            /// </summary>
-            /// <param name='operations'>
-            /// The operations group for this extension method.
-            /// </param>
-            /// <param name='resourceGroupName'>
-            /// Name of the Resource group within the Azure subscription.
-            /// </param>
-            /// <param name='namespaceName'>
-            /// The namespace name
-            /// </param>
-            public static IPage<Eventhub> ListByNamespace(this IEventHubsOperations operations, string resourceGroupName, string namespaceName)
```

Comment:
Why these operations are removed? This will be a breaking change. Could we get these back?